### ISSUE TITLE
Is it possible to return the detected Language in the Elasticsearch API

### ISSUE THREAD

**Issue by 000panther** [jprante/elasticsearch-langdetect#31]
In all the examples, the detected language is only queried, never returned. In my use-case I would like to classify documents as english or german, persist them, later another job works over the data and filters the data into language specific elasticsearch databases.

First Question: Is there always one specific language assigned, or is it possible that multiple languages are assigned?
Second Question: Can I somehow persist the detected language in the documents or return it when fetching the documents from elasticsearch? 

If everything else fails, I could use the language analyzer api your script provides and send the text , or use 2 queries to fetch the documents (one for en one for de) , but I wanted to know beforehand if I am doing something wrong or misunderstanding anything.

**Comment by srmelody** [comment]
I'm not sure if @jprante is watching this issue or if @hobbysoldat found a way to make this work, but I agree that this is pretty useful functionality and seems like it is somewhere in the document.

**Comment by jprante** [comment]
I'm a bit confused, because I think the answers are at hand: yes, langdetect can detect multiple languages, and yes, of course the language code is persisted and can be returned, for instace if the field type is set to `store: true`
Seems that I have to add more examples when I will release the plugin for ES 2.0.

**Comment by srmelody** [comment]
That would be great @jprante.  It is unclear to me how to return the language code when issuing a query.  The examples of searching fields based on the language code are excellent and work great for me.  Thanks again for this amazing plugin (and all the other amazing elasticsearch plugins you provide)! :smile:

**Comment by jprante** [comment]
I added an example to demonstrate the retrieval of the detected language codes.

Also, version 2.0.0.0 is now released, for ES 2.0

**Comment by cosmin-marginean** [comment]
Hi @jprante 
I have a similar question and I actually couldn't find some recent samples for this.

We basically have a field like the following

```
...
              "contentLanguage": {
                    "type": "langdetect"
                }
...
```

Which we then index with this for example

```
...
"contentLanguage": "Ja, genau"
...
```

What we're interested in is a) filtering by language, which I found samples for and b) return at search time the value for the detected language, which in turn is what I'm struggling to identify.

Thank you
Cos
